paddle/fluid/inference/api/demo_ci/vis_demo.cc

```diff
@@ -100,20 +101,34 @@ void CheckOutput(const std::string& referfile, const PaddleTensor& output) {
 /*
  * Use the native fluid engine to inference the demo.
  */
-void Main(bool use_gpu) {
-  NativeConfig config;
-  config.param_file = FLAGS_modeldir + "/__params__";
-  config.prog_file = FLAGS_modeldir + "/__model__";
-  config.use_gpu = use_gpu;
-  config.device = 0;
-  if (FLAGS_use_gpu) {
+void Main(bool use_gpu, bool use_trt) {
+  std::unique_ptr<PaddlePredictor> predictor;
+  if (!use_trt) {
+    NativeConfig config;
+    config.param_file = FLAGS_modeldir + "/__params__";
+    config.prog_file = FLAGS_modeldir + "/__model__";
+    config.use_gpu = use_gpu;
+    config.device = 0;
+    if (FLAGS_use_gpu) {
+      config.fraction_of_gpu_memory = 0.1;  // set by yourself
+    }
+
+    VLOG(3) << "init predictor";
+    predictor =
+        CreatePaddlePredictor<NativeConfig, PaddleEngineKind::kNative>(config);
+  } else {
+    paddle::contrib::MixedRTConfig config;
+    config.param_file = FLAGS_modeldir + "/__params__";
+    config.prog_file = FLAGS_modeldir + "/__model__";
+    config.use_gpu = true;
+    config.device = 0;
+    config.max_batch_size = 1;
     config.fraction_of_gpu_memory = 0.1;  // set by yourself
+    predictor =
+        CreatePaddlePredictor<paddle::contrib::MixedRTConfig,
+                              PaddleEngineKind::kAutoMixedTensorRT>(config);
```

Comment:
Here change to 

predictor =
        CreatePaddlePredictor<paddle::contrib::MixedRTConfig>(config);

Reply:
Done